website/docs/language/state/remote-state-data.mdx

```diff
@@ -13,6 +13,12 @@ You can use the `terraform_remote_state` data source without requiring or config
 
 :::warning
 
+`terraform_remote_state` does not respect the sensitivity of source state snapshots. Use caution when referencing state with sensitive root module output values.
```

Comment:
nit:
```suggestion
`terraform_remote_state` does not respect the sensitivity of source state snapshots. Use with caution when referencing state with sensitive root module output values.
```

Reply:
I think both forms are correct here, telling someone to "use caution" and "use with caution" are both valid.

I would actually propose a third alternative to make things worse: `Please be cautious when referencing....`

Reply:
I dislike the "Please", I want the message to be a call to action and not gentle.

Reply:
FWIW, the prevailing writing style for the docs avoids things like "Please" and prefers to just directly assert facts, instructions, or recommendations, and so I agree with the comment above and would personally broaden that to _all_ of our documentation, rather than just this one example.